src/Data/Text/Rope.hs

```diff
@@ -158,7 +156,8 @@ length = _metricsCharLen . metrics
 newlines :: Rope -> Word
 newlines = _metricsNewlines . metrics
 
--- | Measure text length as an amount of lines and columns, O(1).
+-- | Measure text length as an amount of lines and columns.
+-- Time is linear in the length of the last line.
```

Comment:
If we want to go with this change, please reflect it in the changelog.

Reply:
Done